README.md

```diff
@@ -19,6 +19,21 @@ This action generates a changelog based on your Git history using [git-cliff](ht
 
 - `OUTPUT`: Output file. (Default: `"git-cliff/CHANGELOG.md"`)
 
+### Important Notes
+- It is important to checkout the repositories as:
+  
+  ```yaml
+  - name: Checkout
+    uses: actions/checkout@v4
+    with:
+      fetch-depth: 0
+  ```
+
+  `fetch-depth: 0` the is key argument to checkout.
+  
+  Otherwise, you might end-up getting empty changelogs or following errors depending on arguments passed to `git-cliff`:\
+  `ERROR git_cliff > Git error: revspec '<tag/sha>' not found; class=Reference (4); code=NotFound (-3)`
```

Comment:
I feel like giving an example of the error is a bit too verbose. Can you remove it?
```suggestion
  Otherwise, you might end-up getting empty changelogs or following errors depending on arguments passed to `git-cliff`
```

Reply:
I wanted to add the error intentionally because: When I get the error I tried to google the error and found nothing useful, then went over both `git-cliff` and `git-cliff-action` repositories issues and search the error there too, so that I'd get a straight forward solution. But if you think it shouldn't be in documentation, it still makes sense.